apps/next/src/components/SwitchCard/SwitchCard.tsx

```diff
@@ -3,40 +3,56 @@ import {
   styled,
   Switch,
   Typography,
-  TypographyProps,
+  TypographyVariant,
 } from '@avalabs/k2-alpine';
 
 import { Card } from '../Card';
+import { useTheme } from '@emotion/react';
 
 type Orientation = 'horizontal' | 'vertical';
 
 interface SwitchCardProps {
   title: string;
-  titleProps?: TypographyProps;
   description: string;
   orientation?: Orientation;
   checked: boolean;
   disabled?: boolean;
   onChange: () => void;
+  titleSize: 'small' | 'large';
 }
 
-const defaultTitleProps: TypographyProps = {
-  variant: 'h6',
-};
-
 export const SwitchCard = ({
   title,
   description,
   orientation = 'vertical',
   checked,
   disabled,
   onChange,
-  titleProps = defaultTitleProps,
+  titleSize = 'small',
 }: SwitchCardProps) => {
+  const theme = useTheme();
+  const titleStyles = {
+    small: {
+      variant: 'subtitle3' as TypographyVariant,
+      // TODO: fix it after a proper k2 release
+      fontWeight: 600,
+    },
+    large: {
+      variant: 'h5' as TypographyVariant,
+      fontWeight: theme.typography.fontWeightBold,
+    },
+  };
```

Comment:
Since this is constant, can it be moved outside of the component body? Instead of `theme.typograpy.fontWeightBold`, just `fontWeightBold` should work.